homeassistant/components/airvisual/__init__.py

```diff
@@ -275,7 +275,7 @@ async def async_update_data():
     return True
 
 
-async def async_migrate_entry(hass, config_entry):
+async def async_migrate_entry(hass, config_entry) -> bool:
```

Comment:
We shouldn't mix random typing into this PR; that should be saved for its own PR.